### ISSUE TITLE
[Modal] Blur doesn't work in modal without background

### ISSUE THREAD

**Issue by staniel359** [fomantic/Fomantic-UI#3061]
# Bug Report
Blur doesn't work in modal without background.
The element is just transparent like its parent.

But somehow it works in transparent segment. (See `Expected result`).

## Steps to reproduce

1. Create a modal with `background: none`
2. Inside `<div class="content">` place a child element with `backdrop-filter: blur(0.5rem)` 

## Expected result
To have a blur.

![image](https://github.com/fomantic/Fomantic-UI/assets/18214933/418d6305-beff-48d2-a783-92472928b129)

## Actual result
![image](https://github.com/fomantic/Fomantic-UI/assets/18214933/abc5bc19-2414-4f23-9903-d9cd01c2b3ac)

## Testcase
See `Steps to reproduce`

## Screenshot (if possible)
See above.

## Version
2.9.3

**Comment by lubber-de** [comment]
If you use the backdrop filter on the dropdown menu and also have the dropdown menu background set to "none", it seems like what you want to achive.
Otherwise please adjust the jsfiddle according to your needs. Much easier than guessing your codebase :wink:
https://jsfiddle.net/lubber/o0cbxv7a/11/
![image](https://github.com/fomantic/Fomantic-UI/assets/18379884/e3b8120f-a75a-429b-892e-797157dfdfe7)

**Comment by staniel359** [comment]
@lubber-de When I add
```css
.ui.modal, .content {
  background: none !important;
}
```
it works in Firefox:

![image](https://github.com/fomantic/Fomantic-UI/assets/18214933/75297835-d75c-4544-8209-3ecad49eb898)

but doesn't work in Chrome:

![image](https://github.com/fomantic/Fomantic-UI/assets/18214933/c8c132fd-6a1f-4bb2-9f8a-4d3d662c51a5)

**Comment by staniel359** [comment]
@lubber-de Also weird thing happens when I toggle `will-change` property:

[Screencast from 24-06-24 14:53:29.webm](https://github.com/fomantic/Fomantic-UI/assets/18214933/773b4c9c-7196-4a64-9d7b-90a41c2f5532)

**Comment by xyloflake** [comment]
This might be the first time I see something working in firefox, but not in chrome XD.

Maybe something deep down has the properties set for [mozilla only props](https://developer.mozilla.org/en-US/docs/Web/CSS/Mozilla_Extensions)? Just suggesting, **might be wrong** since I haven't taken a look.

cc @gamersi

**Comment by gamersi** [comment]
Yeah that's possible. Another possibility is that Blink just screws up

**Comment by xyloflake** [comment]
@lubber-de can you add back `type/bug`? I think this clearly is a bug?

**Comment by lubber-de** [comment]
When you reset `will-change` for the page dimmer, at least the backdrop works for the modal itself in chrome.
See https://jsfiddle.net/lubber/o0cbxv7a/27/
Chrome still bugs on nested backdrop-filter for the dropdown menu inside the modal. Well, the backdrop filter is somewhat applied, but blurs the body instead of the direct modal content parent and only inside the modal content area.
Firefox does this correctly.

Yes, it's a bug, but a bug in chromium, not Fomantic.
https://issues.chromium.org/issues/40835530

**Comment by staniel359** [comment]
Got it, thanks.
